core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -282,8 +307,9 @@ class BrokerToControllerRequestThread(
   config: KafkaConfig,
   time: Time,
   threadName: String,
-  retryTimeoutMs: Long
-) extends InterBrokerSendThread(threadName, networkClient, config.controllerSocketTimeoutMs, time, isInterruptible = false) {
+  networkClientRetryTimeoutMs: Int,
+  requestThreadRetryTimeoutMs: Long
+) extends InterBrokerSendThread(threadName, networkClient, networkClientRetryTimeoutMs, time, isInterruptible = false) {
```

Comment:
Pass in `config.controllerSocketTimeoutMs` for the networkClientRetryTimeoutMs for there to be no change.  That is what we do for all ZK-based broker usage.